examples/Mac/FilterShowcase/sugar.cmake

```diff
@@ -22,5 +22,6 @@ sugar_files(
 
 sugar_files(
     FILTERSHOWCASE_RESOURCES
+    FilterShowcase/en.lproj/MainMenu.xib
```

Comment:
This doesn't have any effects since file already listed in `FILTERSHOWCASE_RESOURCES` by sugar magic:
* https://github.com/hunter-packages/GPUImage/blob/fa4b269ef6bf81e2e8f82213685a318ea8440a42/examples/Mac/FilterShowcase/FilterShowcase/en.lproj/sugar.cmake